spec/controllers/admin/pages/find_all_slices_spec.rb

```diff
@@ -12,11 +12,11 @@
     end
 
     let :id do
-      Moped::BSON::ObjectId.new
+      BSON::ObjectId.new
     end
 
     let :page do
-      double(:page).as_null_object
+      double(:page, as_json: {}).as_null_object
```

Comment:
Commit says:

> This was causing a stack level too deep error.

But it would be good to know why _that_ was happening. What's the root cause?